lib/capistrano/puma/systemd.rb

```diff
@@ -13,7 +13,7 @@ def define_tasks
     def set_defaults
       set_if_empty :puma_systemctl_bin, '/bin/systemctl'
       set_if_empty :puma_service_unit_name, -> { "puma_#{fetch(:application)}_#{fetch(:stage)}" }
-      set_if_empty :puma_service_unit_user, :system
+      set_if_empty :puma_systemctl_user, :system
       set_if_empty :puma_enable_lingering, false
```

Comment:
I also think this variable could default like so:
```suggestion
      set_if_empty :puma_enable_lingering, -> { fetch(:puma_systemctl_user) != :system }
```
...given the likelihood that the deploying user won't have any remaining sessions open after deployment.

Reply:
A default for `puma_lingering_user` wouldn't go amiss either!
```ruby
set_if_empty :puma_lingering_user, -> { fetch(:user) }
```

Reply:
👍🏼 New commit pusshed.